### PR TITLE
Add some docs for the new rule inclusion config

### PR DESCRIPTION
**Pull request by simonbaird**
Preview at https://simonbaird.github.io/ec-policies/ec-policies/policy_configuration.html .

### CODE REVIEW

**Review by zregvart — APPROVED** [review]

**Comment by simonbaird** [comment]
I forgot some parts, in particular the special syntax for tests, and the deprecation of the old field.

**Comment by simonbaird** [comment]
Let's merge.

File: antora/docs/modules/ROOT/templates/policy_configuration.hbs [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+create or manage this configuration manually.
+
+The Enterprise Contract Task and the Enterprise Contract Policy CRD are
+documented elsewhere. _TODO: Where?_
+
+== Configuration Source
+
+When the Enterprise Contract runs it will look for policy configuration in
+`data.config.policy`. The easiest way to populate that data is by adding a JSON
+formatted file in the `data` directory such as `data/config.json`.
```
Comment (lcarva) [inline antora/docs/modules/ROOT/templates/policy_configuration.hbs:21]: The name of the file is irrelevant. The `config` from `data.config.policy` comes from the top-level key in the `config.json` file, called `config`, not the file name.

Just sharing what I learned. I think it is a good idea to stick with a known file name for the sake of convention though.

Comment (simonbaird) [reply antora/docs/modules/ROOT/templates/policy_configuration.hbs:21]: Yeah, you're right. I could maybe explain that to avoid potential confusion.

**Pull request merged**